README.md

```diff
@@ -912,12 +916,12 @@ Sends tokens from the source network to a receiver on the destination network vi
 **Usage:**
 
 ```shell
-npx hardhat send-ccip-tokens --network <sourceNetworkName> 
---pool <sourcePoolAddress> 
---token <sourceTokenAddress> 
---destination-network <destinationNetworkName> 
---receiver <receiverAddress> 
---amount <amountToSend> 
+npx hardhat send-ccip-tokens --network <sourceNetworkName>
```

Comment:
should we make this easy to paste multiline by having a `\` at the end of each line?